.gitignore

```diff
@@ -1,14 +1,13 @@
 node_modules/
-tmp/
```

Comment:
Otherwise `tmp.d.ts` is ignored. If it is still required, `/` maybe prepended to avoid such issues.